.github/workflows/.windows.yml

```diff
@@ -91,7 +91,6 @@ jobs:
           & docker build `
             --build-arg WINDOWS_BASE_IMAGE `
             --build-arg WINDOWS_BASE_IMAGE_TAG `
-            --build-arg GO_VERSION `
```

Comment:
Why are this line and the similar one below being removed? Won't it make the variable set in the `env:` block inert?

Reply:
Hmm good question it's been a long time since I did this. I think this may have been to work around the "go versions don't have `.0` issue, and RC versions are not formatted as SemVer", and "actions/setup-go" only supporting SemVer, thus... could not use the same format.

I can add it back; that said, we don't run a matrix of versions in CI, and we always update both the Dockerfile and the GHA workflows, so in practice it's really redundant, so maybe removing it is the right thing to do?

Reply:
Maybe you're right; I just think we should give that particular change due consideration on its own merits rather than slipping it into this PR.

Reply:
Yes, that makes sense; I removed it from this PR